doc/release/1.16.0-notes.rst

```diff
@@ -37,3 +37,12 @@ Improvements
 
 Changes
 =======
+
+Comparison ufuncs will now error rather than return NotImplemented
+------------------------------------------------------------------
+
+Previously, comparison ufuncs would return `NotImplemented` if their arguments
+had structured dtypes, to help the comparison operators deal with those.  This
```

Comment:
Can you point out that the comparison operators continue to return `NotImplemented` as before?